helpers/peer-frontend/contracts/PeerFrontend.sol

```diff
@@ -176,20 +176,20 @@ contract PeerFrontend {
       uint256(keccak256(abi.encodePacked(
         block.timestamp,
         address(this),
-        _makerToken,
+        _signerToken,
         IPeer(peerContract).tradeWallet(),
-        _takerToken))),
+        _senderToken))),
       block.timestamp + 1,
       Types.Party(
-        address(this), // consumer is acting as the maker in this case
-        _makerToken,
-        makerAmount,
+        address(this), // consumer is acting as the signer in this case
```

Comment:
Comment here mentioned `consumer`, I'd say `peer frontend`

Reply:
And again on line 174 (can't seem to comment on that line sorry)

Reply:
addresses all the uses of consumer in PeerFrontend